tests/c_languages/ClangComplexityBearTest.py

```diff
@@ -59,11 +59,14 @@ def test_output(self):
             end_column=2),)
         expected_result = Result(
             self.bear,
-            "The cyclomatic complexity of function levels(int, int, int) is "
-            "10 which exceeded maximal recommended value of 8.",
+            "This function '{function}' should be simplified. The cyclomatic "
```

Comment:
im sorry, i still disagree on using this. ``This function sum`` sounds bad. Leaving ``Function {function}`` would be waaay better imo